apps/next/src/pages/AccountManagement/components/ImportKeystoreFile/components/KeystoreFilePassword.tsx

```diff
@@ -48,10 +48,11 @@ export const KeystoreFilePassword = ({
       onSuccess: () => {
         capture('KeystoreFileImportSuccess');
         toast.success(t('Successfully imported the keystore file.'));
-        replace('/account-management');
+        // Go back 2 steps: skip import page and add-wallet page
+        go(-2);
```

Comment:
I think it would be better to be more explicit -- navigate directly to `/portfolio`.